modules/core/eks/app.py

```diff
@@ -10,8 +10,17 @@
 module_name = os.getenv("ADDF_MODULE_NAME")
 vpc_id = os.getenv("ADDF_PARAMETER_VPC_ID")  # required
 private_subnet_ids = json.loads(os.getenv("ADDF_PARAMETER_PRIVATE_SUBNET_IDS"))  # required
+custom_subnet_ids = json.loads(os.getenv("ADDF_PARAMETER_CUSTOM_SUBNET_IDS"))
+isolated_subnet_ids = (
+    json.loads(os.getenv("ADDF_PARAMETER_ISOLATED_SUBNET_IDS"))
+    if os.getenv("ADDF_PARAMETER_ISOLATED_SUBNET_IDS")
+    else None
+)  # required only for isolated environments
+eks_version = os.getenv("ADDF_PARAMETER_EKS_VERSION")  # required
 eks_compute_config = json.loads(os.getenv("ADDF_PARAMETER_EKS_COMPUTE"))  # required
 eks_addons_config = json.loads(os.getenv("ADDF_PARAMETER_EKS_ADDONS"))  # required
+if os.getenv("ADDF_PARAMETER_CODEBUILD_SG_ID"):
```

Comment:
L14 and L22 are dependent IMO. Only if the scenario is isolated, then you would need to ask for codebuild security group id, which means the eks api endpoint should be private. You may want to introduce that single check here